### PR TITLE
[unit-tests-only] Add unit tests for OcAppContent component

### PR DESCRIPTION
**Pull request by saw-jan**
## Description
Adds unit  tests for `OcAppContent` component

## Related Issue
- Part of #5234

## Motivation and Context
- increase unit tests coverage

## How Has This Been Tested?
- test environment: local & CI

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:
- [ ] Code changes
- [x] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by kiranparajuli589** [comment]
lgtm 👍🏼

**Review by kiranparajuli589 — COMMENTED** [review]
lgtm 👍🏼

File: packages/web-runtime/tests/unit/components/OcAppContent.spec.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { shallowMount } from '@vue/test-utils'
+import OcAppContent from '../../../src/components/OcAppContent.vue'
+
+const selectors = {
+  appContent: '.oc-app-content'
+}
+
+describe('SkipTo component', () => {
```
Comment (paulcod3) [inline packages/web-runtime/tests/unit/components/OcAppContent.spec.js:8]: i think it should be OcAppContent component

**Review by dpakach — COMMENTED** [review]
lgtm

**Review by pascalwengerter — APPROVED** [review]
Could've been handled in a snapshot test but LGTM

**Pull request merged**